### DISCUSSION TITLE
Scenegraph Inspector missing some nodes

### DISCUSSION THREAD

**Discussion by rrirower** [rokucommunity/vscode-brightscript-language#553] [Q&A] [answered]
While debugging, I noticed that my Grid screen that contains 16 rows (RowList items) only shows 11.  Is this a bug?

**Comment by triwav** [comment, marked as answer]
Hi @rrirower. Any of the RowList items is our best effort at trying to get the children. The logic for that is far from perfect and is pretty complicated. There are some ideas for things we could do in the future to improve it but that's likely a ways off. One suggestion. Sometimes the reason you can't access it is because we don't know what its parent is. In this case those are usually shown at the top level of the node tree. Reading your question again, I'm wondering if perhaps part of the reason you aren't seeing them is because they don't exist yet. If you can not see those components on screen then that could likely be the reason and they just don't exist yet.
